common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4926,9 +4926,14 @@
           "Catalog <catalogName> does not support <operation>."
         ]
       },
+      "CLAUSE_WITH_PIPE_OPERATORS" : {
+        "message" : [
+          "The SQL pipe operator syntax using |> does not support <clauses>."
+        ]
+      },
       "COMBINATION_QUERY_RESULT_CLAUSES" : {
         "message" : [
-          "Combination of ORDER BY/SORT BY/DISTRIBUTE BY/CLUSTER BY."
```

Comment:
looking at the current code change, seems we don't need to update it anymore?

Reply:
Sounds good, this is done.